src/Components/Components/src/Sections/SectionContent.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Sections
+{
+    /// <summary>
+    /// Provides content to <see cref="SectionOutlet"/> components with matching <c>Name</c>s.
```

Comment:
Does this work?

```suggestion
    /// Provides content to <see cref="SectionOutlet"/> components with matching <see cref="SectionContent.Name" />.
```

Not a big deal if not!